aws_xray_sdk/core/patcher.py

```diff
@@ -60,6 +61,9 @@ def _is_valid_import(module):
 
 
 def patch(modules_to_patch, raise_errors=True, ignore_module_patterns=None):
+    enabled = aws_xray_sdk.global_sdk_config.sdk_enabled()
+    if not enabled:
```

Comment:
A debug level log entry is useful to show why patch logic is skipped

Reply:
Done